mars/resource.py

```diff
@@ -391,3 +391,25 @@ def cuda_card_stats() -> List[_cuda_card_stat]:  # pragma: no cover
             )
         )
     return infos
+
+
+def build_band_resources(
```

Comment:
```suggestion
def build_bands_resources(
```

Reply:
It's a collection of (band, resource) pair, I think the `build_band_resources` is good enough. Currently, all the names are `band_resources` not `bands_resources`. @qinxuye @hekaisheng @wjsi What do you think?

Reply:
I think `band_resources` is better for me.